src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.Compilation.targets

```diff
@@ -0,0 +1,163 @@
+<Project>
+  <!--
+    What follows here was copied and modified from the XamlPreCompile target in 
+    Microsoft.CSharp.CurrentVersion.targets
+
+    The XamlPreCompile target must remain identical to
+    the CoreCompile target in Microsoft.CSharp.Core.targets.
+    Any updates to one must be made to the other.
+  -->
+
+
+  <ItemGroup>
+    <!-- Used to creating the final compiled Razor dll -->
+    <_RazorIntermediateAssembly Include="$(IntermediateOutputPath)$(RazorTargetName).dll" />
+    <_RazorIntermediatePdb Condition="'$(_DebugSymbolsProduced)' == 'true'" Include="$(IntermediateOutputPath)$(RazorTargetName).pdb" />
```

Comment:
Isn't `_DebugSymbolsProduced` private? Should we be taking a dependency on something else?

Reply:
We could replicate their check here - https://github.com/Microsoft/msbuild/blob/908cc9ccd4961441628f68e37a148183a87bb067/src/Tasks/Microsoft.Common.CurrentVersion.targets#L146-L153 as part of the condition
`Condition="'$(DebugSymbols)' == 'true' and '$(DebugType) != '' and $(DebugType) != 'none'" and $(DebugType) != 'embedded'"`

while we follow up with them to figure out if they could expose the property publicly.

Reply:
Let's duplicate the check, I think that's fine. Those are all public things.